front/src/components/App/MaintenanceArea/AuctionSitesAddition.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import {existsThisName} from "./Utils";
+import {AuctionSite} from "./AuctionSitesManagement";
+
+export interface AuctionSitesInterface {
+  setAuctionsSites: React.Dispatch<React.SetStateAction<AuctionSite[]>>;
+  auctionsSites: AuctionSite[];
+
+  FormValues?: {
```

Comment:
nazwy zmiennych i pól z małych liter, ponadto raczej nie zagnieżdzajmy obiektów w interejsach, powinno by to wyniesione do osobnego interfejsu